alfresco-java-event-api/alfresco-java-event-api-integration/src/main/java/org/alfresco/event/sdk/integration/transformer/EventGenericTransformer.java

```diff
@@ -36,19 +37,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(EventGenericTransformer.class);
 
-    private final ObjectMapper objectMapper;
-
-    /**
-     * Public constructor.
-     *
-     * @param objectMapper given @{@link ObjectMapper} to use in the JSON transformation process
-     */
-    public EventGenericTransformer(final ObjectMapper objectMapper) {
-        this.objectMapper = Objects.requireNonNull(objectMapper);
-    }
-
     @Override
     public RepoEvent<DataAttributes<Resource>> transform(final String eventJSON) {
+        final ObjectMapper objectMapper = new ObjectMapperFactory().createObjectMapper();
```

Comment:
It's fine to remove it from the constructor parameter, but let's leave it as a field that we either initialise in the constructor or in the definition. I agree on not having it as a static as that can end up as a bottleneck, but let's create one per object vs one per call as it is now.

Reply:
I agree. Changed!